object-group-game/GetConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace object_group_game
+{
+    public static class GetConfiguration
+    {
+        public static string GetSQL()
+        {
+            var someData = new Dictionary<string, string>() { { "ConnectionString", "" } };
+            string path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), @"config.txt");
+            try
+            {
+                using StreamReader sr = new StreamReader(path);
```

Comment:
Check out how to use using statement properly https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement